L/libigc/build_tarballs.jl

```diff
@@ -78,34 +51,29 @@ CMAKE_FLAGS=()
 # Release build for best performance
 CMAKE_FLAGS+=(-DCMAKE_BUILD_TYPE=Release)
 
-# Install things into $prefix, and make sure it knows we're cross-compiling
+# Install things into $prefix
 CMAKE_FLAGS+=(-DCMAKE_INSTALL_PREFIX=${prefix})
-CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING=True)
 
-# Tell LLVM where our pre-built tools are
-CMAKE_FLAGS+=(-DLLVM_CONFIG_PATH=${WORKSPACE}/bootstrap/bin/llvm-config)
-CMAKE_FLAGS+=(-DLLVM_TABLEGEN=${WORKSPACE}/bootstrap/bin/llvm-tblgen)
-CMAKE_FLAGS+=(-DCLANG_TABLEGEN=${WORKSPACE}/bootstrap/bin/clang-tblgen)
-CMAKE_FLAGS+=(-DCLANG_TOOL=${WORKSPACE}/bootstrap/bin/clang)
-
-# Don't have IGC use target Clang
-sed -i 's/add_executable(clang-tool ALIAS clang)/add_executable(clang-tool IMPORTED GLOBAL)\n  set_property(TARGET clang-tool PROPERTY "IMPORTED_LOCATION" "${CLANG_TOOL}")/' IGC/BiFModule/CMakeLists.txt
+# NOTE: igc currently can't cross compile due to a variety of issues:
+# - https://github.com/intel/intel-graphics-compiler/issues/131
+# - https://github.com/intel/opencl-clang/issues/91
+CMAKE_FLAGS+=(-DCMAKE_CROSSCOMPILING:BOOL=OFF)
 
 # Explicitly use our cmake toolchain file
 CMAKE_FLAGS+=(-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN})
 
-mkdir build && cd build
-cmake ${CMAKE_FLAGS[@]} ..
-make -j${nproc}
-make install
+# Silence developer warnings
+CMAKE_FLAGS+=(-Wno-dev)
+
+cmake -B build -S . -GNinja ${CMAKE_FLAGS[@]}
+ninja -C build install
 """
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
     Linux(:i686, libc=:glibc),
     Linux(:x86_64, libc=:glibc),
-    Linux(:x86_64, libc=:musl),
```

Comment:
Shouldn't this work on musl as well? :thinking:

Reply:
Uhm, you're right, I don't see why not. I'll try locally first, but it'd be good to keep the musl patches, indeed.